lib/parser/index.js

```diff
@@ -28,7 +28,8 @@ var $ = HTML.TAG_NAMES,
  */
 var DEFAULT_OPTIONS = {
     locationInfo: false,
-    treeAdapter: defaultTreeAdapter
+    treeAdapter: defaultTreeAdapter,
+    onToken: function () {}
```

Comment:
Didn't really have any, that's why said "might" :) I'm totally fine with hiding it back if you wish.